pyomo/contrib/pyros/util.py

```diff
@@ -279,6 +280,56 @@ def transform_to_standard_form(model):
     return
 
 
+def get_vars_from_constraints(block):
+    """Determine all variables used in constraint expressions in
+    a Block.
+    """
+    seen = set()
+    for constraint in block.component_data_objects(Constraint,
+                                                   descend_into=Block,
+                                                   active=True):
+        for var in EXPR.identify_variables(constraint.expr):
+            if id(var) not in seen:
+                seen.add(id(var))
+                yield var
+
+
+def replace_uncertain_bounds_with_constraints(model, uncertain_params):
+    """
+    For variables of which the bounds are dependent on the parameters
+    in the list `uncertain_params`, remove the bounds and add
+    explicit variable bound inequality constraints.
+
+    :param model: Model in which to make the bounds/constraint replacements
+    :type model: class:`pyomo.core.base.PyomoModel.ConcreteModel`
+    :param uncertain_params: List of uncertain model parameters
+    :type uncertain_params: list
+    """
+    uncertain_param_set = ComponentSet(uncertain_params)
+
+    # component for explicit inequality constraints
+    uncertain_var_bound_constrs = ConstraintList()
+    model.add_component(unique_component_name(model,
+                                              'uncertain_var_bound_cons'),
+                        uncertain_var_bound_constrs)
+
+    for v in get_vars_from_constraints(model):
```

Comment:
Do you want to add constraints for vars in the objective as well? Or just in constraints?

Reply:
Addressed in the subsequent commits.